blueman/Functions.py

```diff
@@ -244,8 +247,8 @@ def create_menuitem_box(text, pixbuf, orientation=Gtk.Orientation.HORIZONTAL, si
     return item_box
 
 
-def create_menuitem(text, pixbuf):
-    box = create_menuitem_box(text, pixbuf)
+def create_menuitem(text, icon_name=None, pixbuf=None):
```

Comment:
I don't see how this can work for the cases in Services.py that provide a pixbuf as second argument. It would be treated as the `icon_name`.